x11-packages/evince/build.sh

```diff
@@ -0,0 +1,15 @@
+TERMUX_PKG_HOMEPAGE=https://wiki.gnome.org/Apps/Evince
+TERMUX_PKG_DESCRIPTION="document viewer for multiple document formats"
+TERMUX_PKG_LICENSE="GPL-2.0"
+TERMUX_PKG_MAINTAINER="@termux"
+_MAJOR_VERSION=43
+TERMUX_PKG_VERSION=${_MAJOR_VERSION}.0
+TERMUX_PKG_SRCURL=https://download.gnome.org/sources/evince/43/evince-43.0.tar.xz
```

Comment:
Please use `$TERMUX_PKG_VERSION`/`$_MAJOR_VERSION` in `TERMUX_PKG_SRCURL`

Reply:
I didn't notice that. Thank you.